src/Compilers/VisualBasic/Portable/Semantics/OverloadResolution.vb

```diff
@@ -444,6 +470,28 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 Return False
             End Function
+
+            Public Overrides ReadOnly Property OverloadResolutionPriority As Integer
+                Get
+                    Return _property.OriginalDefinition.OverloadResolutionPriority
```

Comment:
>OriginalDefinition [](http://example.com/codeflow?start=37&length=18)

Are we testing cases that require `OriginalDefinition` in `OverloadResolutionPriority` or `GetOverloadResolutionPriorityInfo()`?

Reply:
This is not required for correctness. The effect is not observable, other than us doing less work.